config/locales/general.uk.yml

```diff
@@ -237,6 +242,23 @@ uk:
       no_special_notes:     'поки що нотаток немає'
       completed_questionnaires: 'Заповнені опитувальники'
       study_applications: 'Заяви на навчання'
+      registration_time: 'Дата реєстрації'
```

Comment:
@pavel-pasechnik please extract the changes not directly required in the task to a separate PR. It complicates the code review.